README.md

```
@@ -34,12 +34,12 @@ POST http://<host>:3100/mcp               (direct)
 
 ## Config
 
-Environment variables (prefix `SYSLOG_MCP__`, double underscore for nesting):
+Environment variables (prefix `SYSLOG_MCP_`, double underscore for nesting within sections):
 ```bash
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)